python/ray/autoscaler/_private/command_runner.py

```diff
@@ -37,6 +37,7 @@
     os.path.dirname(os.path.abspath(__file__)), "kubernetes/kubectl-rsync.sh")
 MAX_HOME_RETRIES = 3
 HOME_RETRY_DELAY_S = 5
+SSH_RETRY_DELAY_S = 5
```

Comment:
Can we define this in `https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/_private/constants.py`?

Reply:
ok